### PR TITLE
Update SDK for Appwrite 1.0.X

### PR DESCRIPTION
**Pull request by stnguyen90**
## What does this PR do?

Update SDK for Appwrite 1.0.X.

## Test Plan

Manual Test:

```
$ dart main.dart
Running Create Database API
{$id: 632b926327d04fed969f, name: Movies DB, $createdAt: 2022-09-21T22:38:27.163+00:00, $updatedAt: 2022-09-21T22:38:27.163+00:00}
Running List Databases API
{total: 3, databases: [{$id: 6323bde1b647e183440f, name: Default, $createdAt: 2022-09-16T00:05:53.746+00:00, $updatedAt: 2022-09-16T00:05:53.746+00:00}, {$id: 6323bdfff13b52c259be, name: Default, $createdAt: 2022-09-16T00:06:23.988+00:00, $updatedAt: 2022-09-16T00:06:23.988+00:00}, {$id: 632b926327d04fed969f, name: Movies DB, $createdAt: 2022-09-21T22:38:27.163+00:00, $updatedAt: 2022-09-21T22:38:27.163+00:00}]}
Running create collection API
{$id: 632b9263909c752cd841, $createdAt: 2022-09-21T22:38:27.593+00:00, $updatedAt: 2022-09-21T22:38:27.593+00:00, $permissions: [read("any"), create("any"), update("any"), delete("any")], databaseId: 632b926327d04fed969f, name: Movies, enabled: true, documentSecurity: true, attributes: [], indexes: []}
Running list collection API
{$id: 632b9263909c752cd841, $createdAt: 2022-09-21T22:38:27.593+00:00, $updatedAt: 2022-09-21T22:38:27.593+00:00, $permissions: [read("any"), create("any"), update("any"), delete("any")], databaseId: 632b926327d04fed969f, name: Movies, enabled: true, documentSecurity: true, attributes: [{key: name, type: string, status: available, required: true, array: false, size: 60, default: null}, {key: release_year, type: integer, status: available, required: true, array: false, min: -9223372036854775808, max: 9223372036854775807, default: null}], indexes: []}
Running Add Document API
{name: Spider Man, release_year: 1920, $id: 632b9264519a65642501, $permissions: [read("any"), update("any")], $createdAt: 2022-09-21T22:38:28.334+00:00, $updatedAt: 2022-09-21T22:38:28.334+00:00, $collectionId: 632b9263909c752cd841, $databaseId: 632b926327d04fed969f}
Running List Document API
{total: 1, documents: [{$id: 632b9264519a65642501, $collectionId: 632b9263909c752cd841, $databaseId: 632b926327d04fed969f, $createdAt: 2022-09-21T22:38:28.334+00:00, $updatedAt: 2022-09-21T22:38:28.334+00:00, $permissions: [read("any"), update("any")], data: {name: Spider Man, release_year: 1920, $id: 632b9264519a65642501, $createdAt: 2022-09-21T22:38:28.334+00:00, $updatedAt: 2022-09-21T22:38:28.334+00:00, $permissions: [read("any"), update("any")], $collectionId: 632b9263909c752cd841, $databaseId: 632b926327d04fed969f}}]}
Running Delete Document API
Document deleted: 632b926327d04fed969f
Running delete collection API
collection deleted: 632b9263909c752cd841
Running Delete Database API
Running create bucket API
Bucket created: 632b92653c34c3fde7b9
Running list buckets API
Buckets: {$id: 632b92653c34c3fde7b9, $createdAt: 2022-09-21T22:38:29.246+00:00, $updatedAt: 2022-09-21T22:38:29.246+00:00, $permissions: [], fileSecurity: true, name: my awesome bucket, enabled: true, maximumFileSize: 10000000, allowedFileExtensions: [], compression: none, encryption: true, antivirus: true}
Running Upload File API
File uploaded: {$id: 632b9265c1268b1b1acc, bucketId: 632b92653c34c3fde7b9, $createdAt: 2022-09-21T22:38:29.796+00:00, $updatedAt: 2022-09-21T22:38:29.796+00:00, $permissions: [read("any"), update("any")], name: nature.jpg, signature: f1f434b689dd56e5fbd291d479320b9a, mimeType: image/jpeg, sizeOriginal: 16207, chunksTotal: 1, chunksUploaded: 1}
Running Delete File API
File deleted: 632b9265c1268b1b1acc
Running Delete bucket API
Bucket deleted: 632b92653c34c3fde7b9 
Running Create User API
{$id: 632b92664fdf3f1f9f0b, $createdAt: 2022-09-21T22:38:30.789+00:00, $updatedAt: 2022-09-21T22:38:30.789+00:00, name: Some user, password: $argon2id$v=19$m=65536,t=4,p=3$WUNMTzRFeHF6Vjg0QVZkdA$kIcNQ0oa3owJXS2upjFy1UvSrN3tWTqur0Bo2Zx8oEc, hash: argon2, hashOptions: {memoryCost: 2048, timeCost: 4, threads: 3}, registration: 2022-09-21T22:38:30.788+00:00, status: true, passwordUpdate: 2022-09-21T22:38:30.788+00:00, email: 1663799910213@example.com, phone: , emailVerification: false, phoneVerification: false, prefs: {data: {}}}
Running List User API
{total: 1, users: [{$id: 632b92664fdf3f1f9f0b, $createdAt: 2022-09-21T22:38:30.789+00:00, $updatedAt: 2022-09-21T22:38:30.789+00:00, name: Some user, password: $argon2id$v=19$m=65536,t=4,p=3$WUNMTzRFeHF6Vjg0QVZkdA$kIcNQ0oa3owJXS2upjFy1UvSrN3tWTqur0Bo2Zx8oEc, hash: argon2, hashOptions: {memoryCost: 2048, timeCost: 4, threads: 3}, registration: 2022-09-21T22:38:30.788+00:00, status: true, passwordUpdate: 2022-09-21T22:38:30.788+00:00, email: 1663799910213@example.com, phone: , emailVerification: false, phoneVerification: false, prefs: {data: {}}}]}
Running delete user
user deleted
Running Create Function API
{$id: 632b926751f6528a43bc, $createdAt: 2022-09-21T22:38:31.336+00:00, $updatedAt: 2022-09-21T22:38:31.336+00:00, execute: [], name: test function, enabled: true, runtime: php-8.0, deployment: , vars: [], events: [], schedule: , scheduleNext: , schedulePrevious: , timeout: 15}
Running List Functions API
{total: 3, functions: [{$id: maf, $createdAt: 2022-09-15T19:02:51.999+00:00, $updatedAt: 2022-09-15T19:04:20.015+00:00, execute: [users], name: My Awesome Function, enabled: true, runtime: dart-2.16, deployment: , vars: [{$id: 632376ef175e7562c876, $createdAt: 2022-09-15T19:03:11.095+00:00, $updatedAt: 2022-09-15T19:03:11.095+00:00, key: TEST, value: test, functionId: maf}, {$id: 6328e0139e442adb18dc, $createdAt: 2022-09-19T21:33:07.648+00:00, $updatedAt: 2022-09-19T21:33:07.648+00:00, key: asdf, value: asdf, functionId: maf}], events: [], schedule: , scheduleNext: , schedulePrevious: , timeout: 15}, {$id: py, $createdAt: 2022-09-20T17:39:00.774+00:00, $updatedAt: 2022-09-20T17:39:29.862+00:00, execute: [], name: py, enabled: true, runtime: python-3.9, deployment: 6329fabd18d9cf130d37, vars: [], events: [], schedule: , scheduleNext: , schedulePrevious: , timeout: 15}, {$id: 632b926751f6528a43bc, $createdAt: 2022-09-21T22:38:31.336+00:00, $updatedAt: 2022-09-21T22:38:31.336+00:00, execute: [], name: test function, enabled: true, runtime: php-8.0, deployment: , vars: [], events: [], schedule: , scheduleNext: , schedulePrevious: , timeout: 15}]}
Running Delete Function API
Function deleted
```

## Related PRs and Issues

None

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: main.dart [hunk 1]
```diff
@@ -69,31 +74,39 @@ Future<void> createDatabase() async {
...
 Future<void> createCollection() async {
-  final database = Databases(client, databaseId: databaseId);
+  final database = Databases(client);
   print('Running create collection API');
   try {
     final res = await database.createCollection(
-      collectionId: "movies",
-      permission: 'document',
+      databaseId: databaseId,
+      collectionId: ID.custom("movies"),
```
Comment (abnegate) [inline main.dart]: If we use unique here we can run the playground multiple times without issue

```suggestion
      collectionId: ID.unique(),
```

File: main.dart [hunk 2]
```diff
@@ -118,48 +131,61 @@ Future<void> listCollection() async {
...
-    final response = await database.listDocuments(collectionId: collectionId);
+    final response = await database.listDocuments(
+      databaseId: databaseId,
+      collectionId: collectionId,
+    );
     print(response.toMap());
   } on AppwriteException catch (e) {
     print(e.message);
   }
 }
```
Comment (abnegate) [inline main.dart:186]: Can we add a `deleteDoc` function here

File: main.dart [hunk 3]
```diff
@@ -235,7 +269,11 @@ Future<void> createUser(email, password, name) async {
   print('Running Create User API');
   try {
     final response = await users.create(
-        userId: 'unique()', email: email, password: password, name: name);
+      userId: 'unique()',
```
Comment (abnegate) [inline main.dart]:
```suggestion
      userId: ID.unique(),
```

File: main.dart [hunk 4]
```diff
@@ -270,7 +308,7 @@ Future<void> createFunction() async {
   print('Running Create Function API');
   try {
     final res = await functions.create(
-        functionId: 'testfunction',
+        functionId: ID.custom('testfunction'),
```
Comment (abnegate) [inline main.dart]: If we use unique here we can run the playground multiple times without issue

```suggestion
        functionId: ID.unique(),
```

**Review by abnegate — CHANGES_REQUESTED** [review]

File: main.dart (hunk 4 above)
Comment (stnguyen90) [reply main.dart]: Updated!

File: main.dart (hunk 3 above)
Comment (stnguyen90) [reply main.dart]: Updated!

File: main.dart (hunk 2 above)
Comment (stnguyen90) [reply main.dart:186]: Updated!

File: main.dart (hunk 1 above)
Comment (stnguyen90) [reply main.dart]: Updated!

**Review by abnegate — APPROVED** [review]

**Pull request merged**